docs/source/hsm.md

```diff
@@ -89,31 +108,31 @@ in the `signcerts` folder to retrieve the private key from inside the HSM.
 The process for creating the MSP folders will be different depending on if you
 are using a Fabric Certificate Authority (CA) your own CA.
 
-### Using a Fabric CA
+### Using an HSM with a Fabric CA
+
+You can set up a Fabric CA to use an HSM by making the same edits to the CA server configuration file as you would make to a peer or ordering node. Because you can use the Fabric CA to generate keys inside an HSM, the process of creating the local MSP folders is straightforward. Use the following steps:
+
+1. Create an HSM token and point to it in the Fabric CA server configuration file. When the Fabric CA server starts, the HSM described in the configuration file will generate the CA signing certificate and store the private key inside the HSM. If you are not concerned about exposing your CA signing certificate, you can skip this step and only configure an HSM for your peer or ordering nodes.
 
-You can set up a Fabric CA to use an HSM by making the same edits to the
-configuration file as you would make to a peer or ordering node. Because you can
-use Fabric CA to generate keys inside an HSM, the process of creating the local
-MSP folders is straightforward. Use the following steps:
+2. Use the Fabric CA client to register the peer or ordering node identities with your CA.
 
-1. Create an HSM token and point to it in the Fabric CA server configuration
-file. When the Fabric CA server starts, it will generate the CA signing
-certificate inside your HSM. If you are not concerned about exposing your CA
-signing certificate, you can skip this step.
+3. Before you deploy a peer or ordering node with HSM, you need to enroll the node admin, storing its private key in the HSM. Edit the BCCSP section of the Fabric CA client config file or associated environment variables to point to the HSM configuration for your peer or ordering node. In the Fabric CA Client configuration file, replace the default `SW` configuration with the `PKCS11` configuration and providing the values for your own HSM:
 
-2. Use the Fabric CA client to register the peer or ordering node identities
-with your CA.
+  ```
+  bccsp:
+    default: PKCS11
+    pkcs11:
+      Library: /etc/hyperledger/fabric/libsofthsm2.so
+      Pin: 71811222
+      Label: fabric
+      hash: SHA2
+      security: 256
+      Immutable: false
+  ```
 
-3. Edit the Fabric CA client config file or environment variables to use your
-HSM as the crypto service provider. Then for each node, use the Fabric CA client
-to generate the component MSP folder by enrolling against the node identity. The
-enroll command will generate the private key inside your HSM.
+  Then for each node, use the Fabric CA client to generate the peer or ordering node's MSP folder by enrolling against the node identity that you registered in step 2. The enroll command will use the HSM to generate the private key for the peer or ordering node and store it inside the HSM instead of in the /keystore folder of the associated MSP.
```

Comment:
"the HSM" ... similar to my prior comments, I think it would be better to say "the node's HSM", as I was initially confused between the CA's HSM and the node's HSM.